templates/drupal8/files/.lando.upstream.yml

```diff
@@ -0,0 +1,32 @@
+# This file sets some good defaults for local development using this Platform.sh
+# template with Lando.
+#
+# Note that you should not edit this file so it can continue to receive upstream
+# updates. If you wish to change the values below then override them in your
+# normal .lando.yml.
+
+# These both allow you to test this template without needing a site on Platform.sh
+# However you will want to replace them in your .lando.yml
+name: platformsh-drupal8
+recipe: platformsh
+
+config:
+
+  # This section overrides Platform.sh configuration with values that make more
+  # sense for local development.
+  #
+  # Note that "app" is the name of the application defined in your
+  # .platform.app.yaml or applications.yaml.
+  overrides:
+    app:
+      variables:
+        d8config:
+          "system.file:path:temporary": "/tmp"
+        d8settings:
+          "skip_permissions_hardening": 1
+
+
+# These are tools that are commonly used during development for this template.
+tooling:
```

Comment:
Should we also add Drupal Console here?  We install it by default on D8; I have no idea how many people actually use it, though.

Reply:
We've seen decreasing usage over the last year or so but if its in the template then we should have a tooling route for it as well.